static/style/style.scss

```diff
@@ -42,3 +42,14 @@ section {
     max-width: 900px;
   }
 }
+
+body.modal-open {
```

Comment:
should it not be `body .modal-open`?

Reply:
What's the difference again with this kind of selector? haha

Reply:
without the space it would mean `<body class='modal-open'></body>`
with the space it means all elements with the class `modal-open`

Reply:
Ask me why, `body .modal-open` doesn't work. So I'll just keep it like this and if we ever need to use that class on other components we could change it in the future.

Reply:
Ça marche pas si tu met un espace pcq tu ajoute ta classe sur le body, alors que quand tu utilise un espace ça target les components **dans** le body qui ont `.modal-open` et non le body lui même.